import java.io.IOException;


public class DivisionDemo2 {

	int a=20,b=0,c;
	void check()
	{
		try
		{
		c=a/b;
		}
		catch(ArithmeticException|NullPointerException|IOException e )
		{
			System.out.println("exception divide by zero");
		}
	}
	
	public static void main(String[] args) {
		// TODO Auto-generated method stub
		 DivisionDemo2 d=new  DivisionDemo2();
		 d.check();
		 

	}

}
